docker/github/__init__.py

```diff
@@ -145,13 +145,13 @@ async def get_commits_until(self, sha: str, until: str) -> Tuple[List[Commit], b
         commits = await self.get(f"/repos/{self.repo_path}/commits?{sha_arg}")
         GithubError.raise_from_message(commits)
         result = []
-        end_found = False
+        commit_found = False
```

Comment:
Bitte die Umbenennungen in der Zukunft als separate `refactor`-Kommits einfügen. Die inhaltliche Änderungen die eigentlich zu reviewen sind werden dadurch erheblich kleiner und dadurch viel einfacher und schneller nachvollziehbar.